src/nidigital/examples/configure_time_set_and_voltage_levels/nidigital_configure_time_set_and_voltage_levels.py

```diff
@@ -0,0 +1,144 @@
+        self.drive_return = drive_return
+        self.drive_off = drive_off
+        self.strobe_edge = strobe_edge
+
+
+def convert_drive_format(drive_format):
+    converter = {'NR': nidigital.DriveFormat.NR,
+                 'RL': nidigital.DriveFormat.RL,
+                 'RH': nidigital.DriveFormat.RH,
+                 'SBC': nidigital.DriveFormat.SBC}
+    return converter.get(drive_format, None)
+
+
+def example(resource_name, options,
+            channels=None,
+            voltage_config=None,
+            time_set_config=None):
+
+    with nidigital.Session(resource_name=resource_name, options=options) as session:
+
+        dir = os.path.dirname(__file__)
+
+        # Load pin map (.pinmap) created using Digital Pattern Editor
+        pin_map_filename = os.path.join(dir, 'PinMap.pinmap')
+        session.load_pin_map(pin_map_filename)
+
+        # Configure voltage levels and terminal voltage through driver API
+        session.channels[channels].configure_voltage_levels(voltage_config.vil, voltage_config.vih, voltage_config.vol, voltage_config.voh, voltage_config.vterm)
+        if voltage_config.termination_mode == 'High_Z':
+            session.channels[channels].termination_mode = nidigital.TerminationMode.HIGH_Z
+        elif voltage_config.termination_mode == 'Active_Load':
+            session.channels[channels].termination_mode = nidigital.TerminationMode.ACTIVE_LOAD
+            session.channels[channels].configure_active_load_levels(voltage_config.iol, voltage_config.ioh, voltage_config.vcom)
+        else:
+            session.channels[channels].termination_mode = nidigital.TerminationMode.VTERM
+
+        # Configure time set through driver API
+        session.create_time_set(time_set_config.time_set_name)  # Must match time set name in pattern file
+        session.configure_time_set_period(time_set_config.time_set_name, time_set_config.period)
+        session.channels[channels].configure_time_set_drive_edges(time_set_config.time_set_name, convert_drive_format(time_set_config.drive_format),
+                                                                  time_set_config.drive_on, time_set_config.drive_data,
+                                                                  time_set_config.drive_return, time_set_config.drive_off)
+        session.channels[channels].configure_time_set_compare_edges_strobe(time_set_config.time_set_name, time_set_config.strobe_edge)
+
+        # Load the pattern file (.digipat) created using Digital Pattern Editor
+        pattern_filename = os.path.join(dir, 'Pattern.digipat')
+        session.load_pattern(pattern_filename)
+
+        # Burst pattern, blocks until the pattern is done bursting
+        session.burst_pattern(start_label='new_pattern')
+        print('Start bursting pattern')
+
+        # Disconnect all channels using programmable onboard switching
+        session.selected_function = nidigital.SelectedFunction.DISCONNECT
+    print('Done bursting pattern')
```

Comment:
Indent one more level?

Reply:
I put it outside of the session context to indicate that everything is done, maybe it should be updated to "Done bursting pattern, session closed"?